<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subeEkle</class>
 <widget class="QWidget" name="subeEkle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>512</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Şube Ekle</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
 	 background: #3498db;
	border-radius: 18px;
	font-family: Courier New;
	color: #ffffff;
	font-size: 15px;
	padding: 10px 20px 10px 20px;
	text-decoration: none;
    max-width:100px;
	min-width:100px;
	max-height:20px;	 
	min-height:20px;	 
}

QPushButton:hover {
	background: #3cb0fd;
	text-decoration: none;
}
QLineEdit{
	border: 2.5px double #55aaff;
    border-radius: 10px;
    background: rgb(255, 255, 255);
}
QComboBox{
	border: 2.5px double #55aaff;
    border-radius: 10px;
    background: rgb(255, 255, 255);
}
QWidget{
		font-family: Courier New;
}
QLabel{
font:bold 18px 
}
QTableWidget{
	border: 2.5px double #55aaff;
    border-radius: 10px;
    background: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="sehir">
           <property name="text">
            <string>Şehir Ekle</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="lblSehirKod">
             <property name="text">
              <string>Plaka Kod :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lnSehirPlakaKod">
             <property name="minimumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="lblSehirAd">
             <property name="text">
              <string>Şehir Adı :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lnSehirAdi">
             <property name="minimumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="lbl_xKord">
             <property name="text">
              <string>xKoordinat :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ln_xKord">
             <property name="minimumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="lbl_yKord">
             <property name="text">
              <string>yKoordinat :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ln_yKord">
             <property name="minimumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_sehirEkle">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Ekle</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tblSehirListe">
         <property name="minimumSize">
          <size>
           <width>512</width>
           <height>216</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>512</width>
           <height>216</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>Kod</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ad</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>xKoordinat</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>yKoordinat</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="btnSehirTumu">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Tümü</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSehirGuncelle">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Güncelle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSehirSil">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Sil</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="sube">
           <property name="text">
            <string>Şube Ekle</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lblAd">
             <property name="text">
              <string>Şube Adı :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lnAd">
             <property name="minimumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="lblTel">
             <property name="text">
              <string>Şube Tel :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lnTel">
             <property name="minimumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="lblSehir">
             <property name="text">
              <string>Şehir :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbSehir">
             <property name="minimumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="lblMail">
             <property name="text">
              <string>Şube Mail :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lnMail">
             <property name="minimumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>181</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnSubeEkle">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Ekle</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tblSubeListe">
         <property name="minimumSize">
          <size>
           <width>512</width>
           <height>216</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>512</width>
           <height>216</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>Kod</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ad</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tel</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mail</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Şehir</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnSubeTumu">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Tümü</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSubeGuncelle">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Güncelle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSubeSil">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Sil</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QPushButton" name="btnMerkezSube">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Merkez Şube</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGeri">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Geri</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
